File: nodejs/js/fizzbuzz.js
var fizzBuzzComputer = function(number) {
	var fizz = (number % 3) ? '' : 'Fizz';
	var buzz = (number % 5) ? '' : 'Buzz';
	return [fizz + buzz || number, '\n'].join('');
};

function sodaBee(number) {
	var result = '';
	var i;
	for (i = 1; i <= number; i += 1) {
		result += fizzBuzzComputer(i);
	}
	return result;
}

File: python/src/fizzbuzz.py
def sodaBee(ceiling):
    answerSet = [] 
    
    for number in xrange(1, ceiling + 1):
        answer = '' if number % 3 else 'Fizz'
        answer += '' if number % 5 else 'Buzz'
        answer = answer or str(number)
        answerSet.append(answer)
        
    return '\n'.join(answerSet) + '\n'
